superset/assets/javascripts/explorev2/components/Filter.jsx

```diff
@@ -65,13 +84,23 @@ export default class Filter extends React.Component {
             onChange={this.changeOp.bind(this, this.props.filter)}
           />
           <div className="col-lg-6">
-            <input
+            {this.props.renderFilterSelect ?
+            (<SelectField
```

Comment:
looks like this should be indented 2 spaces. might be cleaner to pull this out to another method and call that here `{this.renderFilterFormField()}`